src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java

```diff
@@ -31,7 +32,7 @@ public static boolean shouldReplace() {
 
 		if (!(Minecraft.getMinecraft().currentScreen instanceof GuiEditSign)) return false;
 
-		TileEntitySign tes = ((GuiEditSign) Minecraft.getMinecraft().currentScreen).tileSign;
+		TileEntitySign tes = ((AccessorGuiEditSign) Minecraft.getMinecraft().currentScreen).getTileSign();
```

Comment:
rancher boot overlay update PauseChamp

Reply:
we have a rancher boot overlay?

Reply:
also do actual review

Reply:
> we have a rancher boot overlay?

we dont

Reply:
its just a dead old class

Reply:
> also do actual review

i will when I am not in class :bangbang: